orion/pkg/tests_integration/session_integration_test.go

```diff
@@ -0,0 +1,115 @@
+package integration_tests
+
+import (
+	"encoding/json"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/domains"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+	"time"
+)
+
+// Test: Create 3 Sessions, 2 With Same Class Id, and GetAllByClassId()
+func Test_CreateSessions(t *testing.T) {
+	resetTable(t, domains.TABLE_SESSIONS)
```

Comment:
I think we'll also need to resetTable for Classes too. I think this needs to be called after resetting table sessions due to the foreign key constraint. Because sessions depend on classes existing, we can't remove classes first because the sessions table will complain it doesn't have correctly mapped classIds.